it-and-security/teams/workstations.yml

```
@@ -61,3 +61,11 @@ queries:
   - path: ../lib/collect-failed-login-attempts.queries.yml
   - path: ../lib/collect-usb-devices.queries.yml
   - path: ../lib/collect-vs-code-extensions.queries.yml
+  - name: Collect hosts with expired SCEP certs
+    description: "For the following issue: https://github.com/fleetdm/confidential/issues/4518. Returns expiration date for macOS hosts's with expired SCEP certs. If a host has a valid (non-expired) SCEP cert, it won't return any results."
```

```suggestion
    description: "For the following issue: https://github.com/fleetdm/confidential/issues/4518. Returns expiration date for macOS hosts's MDM SCEP certs."
```

this is the current intent of the query, we could change it to behave as originally described.

---

Ah, thanks Roberto. I updated the name of the query to "Collect MDM SCEP cert expiration"